st2client/st2client/base.py

```diff
@@ -225,7 +225,7 @@ def _get_cached_auth_token(self, client, username, password):
         :rtype: ``str``
         """
         if not os.path.isdir(ST2_CONFIG_DIRECTORY):
-            os.makedirs(ST2_CONFIG_DIRECTORY)
+            os.makedirs(ST2_CONFIG_DIRECTORY, mode=0o770)
```

Comment:
Still need to set `setguid` bit. Don't see it anywhere, except of tests setting that bit.

Reply:
Fixed, please re-review when you have time.